pythia/cleaning/tests/test_elo.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import pandas as pd
+from pythia.cleaning import ELO
+from pathlib import Path
+from pythia.seo import Sunspotter
+from sunpy.util import SunpyUserWarning
+
+path = Path(__file__).resolve().parent.parent.parent.parent / "data/elo_test"
+
+
+@pytest.fixture
+def elo():
+    sunspotter = Sunspotter(timesfits=path / "../all_clear/lookup_timesfits.csv",
+                            properties=path / "../all_clear/lookup_properties.csv",
```

Comment:
I wouldn't use the original dataset as for tests..

Reply:
#42